src/samples/coap/simple-server.c

```diff
@@ -210,6 +210,7 @@ light_method_get(const struct sol_coap_resource *resource, struct sol_coap_packe
 }
 
 static struct sol_coap_resource light = {
+    .api_version = SOL_COAP_RESOURCE_API_VERSION,
```

Comment:
Looks like oicgen.py don't fill this struct by itself, it uses sol_oic_device_definition_register_resource_type() that calls create_coap_resource().

I'm adding the version on this point, so we should be good about it.

Am I missing any direct usage on oicgen.py?